Lib9c/Action/BattleArena.cs

```diff
@@ -24,6 +24,7 @@ namespace Nekoyume.Action
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/1663
     /// Hard forked at https://github.com/planetarium/lib9c/pull/1649
+    /// Updated at https://github.com/planetarium/lib9c/pull/1679
```

Comment:
Does this `battle_arena8` not require to hard fork?
I guess the `ArenaSimulator` need to change to  `ArenaSimulatorV2` like the `battle_arena7`.

Reply:
`BattleArena8` is not released on mainnet yet.